src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -799,7 +799,18 @@ namespace winrt::TerminalApp::implementation
 
         // Give term control a child of the settings so that any overrides go in the child
         // This way, when we do a settings reload we just update the parent and the overrides remain
-        TermControl term{ *(winrt::get_self<TerminalSettings>(settings)->CreateChild()), connection };
+        const auto child = winrt::get_self<TerminalSettings>(settings)->CreateChild();
+        TermControl term{ *child, connection };
+
+        // Check if the profile defines an unfocusedAppearance
+        const auto profile = _settings.FindProfile(profileGuid);
+        if (profile.UnfocusedAppearance())
+        {
+            // Now, make a grandchild for the unfocused appearance
+            const auto grandchild = child->CreateChild();
+            grandchild->ApplyAppearanceSettings(profile.UnfocusedAppearance(), _settings.GlobalSettings().ColorSchemes());
+            term.UnfocusedAppearance(*grandchild);
+        }
```

Comment:
Idk what Dustin wants in #9296, but could you basically shoehorn that change into here? I see you set up the unfocused appearance a bit later in this file too.

Reply:
I'll get to this once that merges!